script/debug

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
In the README, we've said:

```bash
./script/start --debug
```

Which implies this extra debug script isn't needed?